src/project-tests/pomodoro-clock-tests.js

```diff
@@ -57,18 +57,24 @@ export default function createPomodoroClockTests() {
     return observer;
   }
 
-  // We "Hack" the global setTimeout and setInterval functions so time elapses faster (delay is forced to 30ms)
-  // Note: we should consider putting these hacks in the beforeEach function so every timed test can be done in less time
-  // The problem is that we still don't know if it's acceptable to use this hack, because it implies forcing the campers to use setTimeout and setInterval functions to measure time in their pomodoro.
+  // We "Hack" the global setTimeout and setInterval functions so time elapses 
+  // faster (delay is forced to 30ms)
+  // Note: we should consider putting these hacks in the beforeEach function 
+  // so every timed test can be done in less time
+  // The problem is that we still don't know if it's acceptable to use this 
+  // hack, because it implies forcing the campers to use setTimeout and 
+  // setInterval functions to measure time in their pomodoro.
   const savedSetTimeout = window.setTimeout;
   const savedSetInterval = window.setInterval;
 
   function hackGlobalTimerFunctions() {
     window.setTimeout = (fun, delay) => {
-      return savedSetTimeout(fun, 30);
+      delay = 30;
+      return savedSetTimeout(fun, delay);
     };
     window.setInterval = (fun, delay) => {
-      return savedSetInterval(fun, 30);
+      delay = 30;
+      return savedSetInterval(fun, delay);
```

Comment:
delay param was unused, so used it in order to disable eslint warning.

Reply:
You should just remove the parameter. Javascript is flexible like that, as long as the unused parameters are at the end. If you find yourself in a situation with an unused param at the beginning, you can tell ESLint to ignore it like this: https://eslint.org/docs/rules/no-unused-vars#args